ansible-inventory-origin-15-aio

```diff
@@ -0,0 +1,24 @@
+
+[OSEv3:children]
+nodes
+masters
+
+[OSEv3:vars]
+ansible_connection=local
```

Comment:
Yes.  Unfortunately, openshift-ansible will still attempt to use ssh to the local machine, even when using ansible_connection=local.  I have not yet been able to figure out how to work around this.